pkg/util/maps/string.go

```diff
@@ -1,29 +0,0 @@
-/*
```

Comment:
I would be happy to hear suggestions from the community on whether the current file (package) should be deleted or retained.